### DISCUSSION TITLE
supabase functions - change description

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#7670] [Questions]
Hi Community,

Is there a way to change the description of the function in the studio? Basically the invoke part from the image below

![image](https://user-images.githubusercontent.com/62215539/178641579-d112d0c6-94af-41c9-8163-0cfdd6d5508f.png)

Thanks
Bharat

**Comment by Hallidayo** [comment]
Hi @rbkayz, What do you mean by the description?

**Reply by rbkayz** [reply]
Hello, The API given under invoke your function. How can I add the params of the function here

**Reply by Hallidayo** [reply]
In the `--data` section you should be able to enter parameters in here like so:

--data '{ "param1":"hello", "param2": "world" }'

Then in your function you can use the `req` param to get the values of the parameters that you are passing in and then use these values to execute what you need to do.
